### PR TITLE
Refactor addRemoteDevice

### PR DESCRIPTION
**Pull request by nmattia**
This updates `addRemoteDevice` to be more structured like other pages (captcha, register) and use more idiomatic lit.

### CODE REVIEW

File: src/frontend/src/flows/addDevice/welcomeView/index.ts [hunk 1]
```diff
@@ -3,10 +3,16 @@ import { registerTentativeDevice } from "./registerTentativeDevice";
...
+  onContinue,
+  userNumber,
+}: {
+  onContinue: (userNumber: bigint) => void;
+  userNumber?: bigint;
+}) => {
   const anchorInput = mkAnchorInput({
     userNumber,
-    onSubmit: (userNumber) => registerTentativeDevice(userNumber, connection),
+    onSubmit: onContinue,
```
Comment (peterpeterparker) [inline src/frontend/src/flows/addDevice/welcomeView/index.ts:15]: ultra nth: you can rename when you deconstruct

```
const addRemoteDeviceTemplate = ({
  onContinue: onSubmit,
  userNumber,
}: {
  const anchorInput = mkAnchorInput({
    userNumber,
    onSubmit
```

File: src/frontend/src/showcase.ts [hunk 2]
```diff
@@ -227,7 +227,11 @@ const iiPages: Record<string, () => void> = {
   chooseDeviceAddFlow: () => chooseDeviceAddFlow(),
   renderPollForTentativeDevicePage: () =>
     renderPollForTentativeDevicePage(userNumber),
-  addRemoteDevice: () => addRemoteDevice(dummyConnection),
+  addRemoteDevice: () =>
+    addRemoteDevicePage({
```
Comment (peterpeterparker) [inline src/frontend/src/showcase.ts:231]: you can safely ignore the promise result here?

**Review by peterpeterparker — APPROVED** [review]
LGTM, minor comments

File: src/frontend/src/showcase.ts (hunk 2 above)
Comment (nmattia) [reply src/frontend/src/showcase.ts:231]: yes, it's just a showcase! the result is meaningless

File: src/frontend/src/flows/addDevice/welcomeView/index.ts (hunk 1 above)
Comment (nmattia) [reply src/frontend/src/flows/addDevice/welcomeView/index.ts]: Thanks! added

**Pull request merged**